tests/Http/Middleware/AdminMiddlewareTest.php

```diff
@@ -31,7 +31,7 @@ public function non_admin_are_redirected()
     }
 
     /** @test */
-    public function admin_are_redirected()
+    public function admin_can_access_create_page_redirected()
```

Comment:
This one is a generic test so that is why in the set up I created a generic route. You do not need to include the celeb route in here 😄

Reply:
I have updated the main so this should be now fixed